lib/gis/parser.c

```diff
@@ -547,10 +546,19 @@ int G_parser(int argc, char **argv)
                 snprintf(buff, sizeof(buff), "GRASS_VERBOSE=%d",
```

Comment:
With _G_set_verbose()_ added, may this _snprintf_ + _putenv_ blocks be removed?

When i ran some local tests, it does not seem to have any effect after the first time such a block had run, e.g. if users provide both ` --v` and ` --pp`.

Reply:
> With _G_set_verbose()_ added, may this _snprintf_ + _putenv_ blocks be removed?

I believe it can, `G_set_verbose()` is the dedicated API for just this.